apps/OboeTester/app/src/main/java/com/mobileer/oboetester/TestOutputActivity.java

```diff
@@ -134,6 +139,56 @@ public void onClick(View v) {
                 flushFromFrame();
             }
         });
+
+        mFallbackModeSpinner = (Spinner) findViewById(R.id.fallbackModeSpinner);
+        mFallbackModeTextView = (TextView) findViewById(R.id.fallbackModeText);
+        mStretchModeSpinner = (Spinner) findViewById(R.id.stretchModeSpinner);
+        mStretchModeTextView = (TextView) findViewById(R.id.stretchModeText);
+        mPitchEditText = (EditText) findViewById(R.id.pitchEditText);
+        mPitchTextView = (TextView) findViewById(R.id.pitchText);
+        mSpeedEditText = (EditText) findViewById(R.id.speedEditText);
+        mSpeedTextView = (TextView) findViewById(R.id.speedText);
+        mSetPlaybackParametersButton = (Button) findViewById(R.id.setPlaybackParametersButton);
+        mSetPlaybackParametersButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                setPlaybackParameters();
+            }
+        });
+    }
+
+    private void setPlaybackParameters() {
```

Comment:
What happens when we close the stream? Should we reset the UI?

Alternatively, should we only show the UI after we open the stream and it's not closed?